README.md

```diff
@@ -109,6 +109,7 @@ official compatibility chart below.
 Refer to `kubernetes-deploy --help` for the authoritative set of options.
 
 - `--template-dir=DIR`: Used to set the deploy directory. Set `$ENVIRONMENT` instead to use `config/deploy/$ENVIRONMENT`. This flag also supports reading from STDIN. You can do this by using `--template-dir=-`. Example: `cat templates_from_stdin/*.yml | kubernetes-deploy ns ctx --template-dir=-`.
+- (alpha feature) `-f [PATHS]`: Accepts a comma-separated list of directories and/or filenames to specify the set of directories/files that will be deployed (use `-` to read from STDIN). Cannot be combined with `--template-dir`. Example: `cat templates_from_stdin/*.yml | kubernetes-deploy ns ctx -f -,path/to/dir,path/to/file.yml`
```

Comment:
I believe both ways should work. AFAIK kubectl -f accepts both multiple invocations and a list of filenames/dirs. The same behaviour should work here as well, since invoking `-f` is a purely additive affair

Reply:
🤦‍♂ moment for me. I wasn't using `+=` when parsing the options, instead using raw assignment, so of course it would never work. Changing to increment-operator makes multiple invocations of `-f` now work